store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/AdminGetAllGraphInfoTest.java

```diff
@@ -39,7 +40,7 @@
 
     private static final String ADMIN_AUTH = "AdminAuth";
     private static final User ADMIN_USER = new User("adminUser", null, Sets.newHashSet(ADMIN_AUTH));
-    private static final Class CURRENT_CLASS = new Object() { }.getClass().getEnclosingClass();
+    private static final Class CURRENT_CLASS = new Object() {}.getClass().getEnclosingClass();
     private static final AccumuloProperties PROPERTIES =
             AccumuloProperties.loadStoreProperties(StreamUtil.openStream(CURRENT_CLASS, "properties/singleUseAccumuloStore.properties"));
```

Comment:
I'd just drop it in favour of AdminGetAllGraphInfo.class or make it a private field and use getClass()

```suggestion
    private static final AccumuloProperties PROPERTIES =
            AccumuloProperties.loadStoreProperties(StreamUtil.openStream(AdminGetAllGraphInfoTest.class, "properties/singleUseAccumuloStore.properties"));
```